torch/csrc/cuda/Module.cpp

```diff
@@ -99,7 +95,7 @@ PyObject* THCPModule_setDevice_wrap(PyObject* self, PyObject* arg) {
 PyObject* THCPModule_exchangeDevice(PyObject* self, PyObject* arg) {
   HANDLE_TH_ERRORS
   TORCH_CHECK(THPUtils_checkLong(arg), "invalid argument to exchangeDevice");
-  int64_t device = THPUtils_unpackLong(arg);
+  int device = THPUtils_unpackInt(arg);
```

Comment:
Is that automatic clang tidy update? That doesn't look consistent with other device args

If there is a mix of automatic skips and manual changes, could you split them up to ease review?